samples/sdl_gamecontroller/main.c

```diff
@@ -0,0 +1,46 @@
+#include <xboxrt/debug.h>
+#include <pbkit/pbkit.h>
+#include <hal/xbox.h>
+#include <assert.h>
+#include <SDL.h>
+#include <stdbool.h>
+
+int main() {
+  int ret = pb_init();
+  if (ret != 0) {
+    XSleep(2000);
+    return -1;
+  }
+
+  pb_show_debug_screen();
+
+  SDL_InitSubSystem(SDL_INIT_GAMECONTROLLER);
+
+  SDL_GameController *gameController;
+  gameController = SDL_GameControllerOpen(0);
+
+
+  debugPrint("Num joysticks      : %02i\n", SDL_NumJoysticks());
+  debugPrint("GameController name: %s\n", SDL_GameControllerName(gameController));
+  debugPrint("Is GameController  : %02i\n", SDL_IsGameController(0));
+  debugPrint("GameController Map : %s\n", SDL_GameControllerMapping(gameController));
+  debugPrint("GameControllerEvent: %02i\n", SDL_GameControllerEventState(SDL_QUERY));
+
+  debugPrint("\n");
+  assert(SDL_IsGameController(0));
+
+  SDL_Event event;  
+  while (true) {
+    while (SDL_PollEvent(&event)) {
+      if (event.type == SDL_CONTROLLERBUTTONDOWN) {
+        debugPrint("Button \"%s\" pressed\n", SDL_GameControllerGetStringForButton((SDL_GameControllerButton)event.cbutton.button));
+      } else if (event.type == SDL_CONTROLLERAXISMOTION) {
+        debugPrint("Axis %s moved, new value: %i\n", SDL_GameControllerGetStringForAxis((SDL_GameControllerAxis)event.caxis.axis), event.caxis.value);
+      }
+    }
+  }
+
+  pb_kill();
```

Comment:
If we are showing destruction of objects, there's probably stuff to do for SDL?

Reply:
Thanks, taken care of. :+1: